lib/Service/SynchronizationService.php

```diff
@@ -2243,14 +2241,14 @@ private function processWriteFileRule(Rule $rule, array $data, string $objectId,
                 $openRegisters->setSchema($schemaId);
 
                 try {
-                    // Write file with OpenRegister ObjectService.
-                    $objectService = $this->containerInterface->get('OCA\OpenRegister\Service\ObjectService');
-                    $file = $objectService->addFile(object: $objectId, fileName: $fileName, base64Content: $content);
-
                     $tags = array_merge($config['tags'] ?? [], ["object:$objectId"]);
-                    if ($file instanceof File === true) {
-                        $this->attachTagsToFile(fileId: $file->getId(), tags: $tags);
-                    }
+
+                    $objectService = $this->containerInterface->get('OCA\OpenRegister\Service\ObjectService');
+                    $objectEntity = $objectService->findByUuid(uuid: $objectId);
+                    
+                    // Write file with OpenRegister ObjectService.
+                    $fileService = $this->containerInterface->get('OCA\OpenRegister\Service\FileService');
```

Comment:
What if OpenConnector runs without OpenRegister installed. Shouldn't we catch an exception here?